setup.py

```diff
@@ -7,7 +7,7 @@
     long_description = f.read()
 
 install_requires = [
-    "xarray>=0.14.1",
+    "xarray>=0.17.0",
     "dask",
     "numpy",
     "future",
```

Comment:
Is this still needed? I thought it was only needed for applications that wanted to support Python 2 & Python 3

Reply:
I think you are right. I just removed it. Lets see if that upsets the CI.